cfgov/jinja2/v1/_includes/templates/streamfield-sidefoot.html

```diff
@@ -1,23 +1,18 @@
-{% import 'molecules/related-posts.html' as related_posts with context %}
 {% import 'molecules/related-metadata.html' as related_metadata with context %}
 {% import 'molecules/social-media.html' as social_media with context %}
 
 
 {% macro render(streamfield, half_width=false) %}
     {% for block in streamfield %}
         {% if 'related_posts' in block.block_type %}
-            {% set posts = page.related_posts(block) %}
-            {% if posts %}
-                <div class="block {{ 'block__flush-top' if loop.index == 1 else '' -}}">
-                    {{ related_posts.render(posts, block, half_width) }}
-                </div>
-            {% endif %}
+            <div class="block{{ ' block__flush-top' if loop.first else '' }}">
+            {% include_block block %}
```

Comment:
Wish I had noticed when I reviewed this a year and a half ago that the closing `</div>` went missing here. We've had improperly nested sidebar tags on pages with Related Posts ever since 😞